consensus/enclave/impl/src/lib.rs

```diff
@@ -2340,6 +2341,330 @@ mod tests {
+                vec![governor_2.public_key(), governor_3.public_key()],
+                2,
+            )],
+            2,
+        );
+        let governors_map =
+            GovernorsMap::try_from_iter([(token_id1, governors_signer_set)]).unwrap();
+
+        let minter_1 = Ed25519Pair::from_random(&mut rng);
+        let minter_2 = Ed25519Pair::from_random(&mut rng);
+        let minter_3 = Ed25519Pair::from_random(&mut rng);
+
+        let minters_signer_set = SignerSet::new_with_multi(
+            vec![minter_1.public_key()],
+            vec![SignerSet::new(
+                vec![minter_2.public_key(), minter_3.public_key()],
+                2,
+            )],
+            2,
+        );
+
+        let mut nonce: Vec<u8> = vec![0u8; NONCE_LENGTH];
+        rng.fill_bytes(&mut nonce);
+
+        let mint_config_tx = {
+            let prefix = MintConfigTxPrefix {
+                token_id: *token_id1,
+                configs: vec![MintConfig {
+                    token_id: *token_id1,
+                    signer_set: minters_signer_set,
+                    mint_limit: 10000,
+                }],
+                nonce,
+                tombstone_block: 2,
+                total_mint_limit: 10000,
+            };
+            let message = prefix.hash();
+            let signature = MultiSig::new(vec![
+                governor_1.sign(message.as_ref()),
+                governor_2.sign(message.as_ref()),
+                governor_3.sign(message.as_ref()),
+            ]);
+            MintConfigTx { prefix, signature }
+        };
+
+        let recipient1 = AccountKey::random(&mut rng);
+
+        let mint_tx = create_mint_tx_to_recipient(
+            token_id1,
+            &[minter_1, minter_2, minter_3],
+            12,
+            &recipient1.default_subaddress(),
+            &mut rng,
+        );
+
+        let enclave = SgxConsensusEnclave::new(logger.clone());
+        let blockchain_config = BlockchainConfig {
+            block_version,
+            governors_map: governors_map.clone(),
+            governors_signature: sign_governors_map(&governors_map),
+            ..Default::default()
+        };
+        enclave
+            .enclave_init(
+                &Default::default(),
+                &Default::default(),
+                &None,
+                blockchain_config,
+            )
+            .unwrap();
+
+        // Initialize a ledger.
+        let sender = AccountKey::random(&mut rng);
+        let mut ledger = create_ledger();
+
+        // We want the next block that gets appended to the ledger to exceed the
+        // tombstone limit of the mint config tx, since we want to make sure
+        // that minting that relies on an old MintConfigTx (one that is past
+        // its tombstone block) still validate and mint successfully.
+        let n_blocks = mint_config_tx.prefix.tombstone_block;
+        initialize_ledger(block_version, &mut ledger, n_blocks, &sender, &mut rng);
```

Comment:
:wrench: I'm guessing this is copy pasta from other tests here.  Suggest this test focus on "one" thing and that is the nested multisig.
```suggestion
        initialize_ledger(block_version, &mut ledger, 1, &sender, &mut rng);
```

Reply:
If the ledger doesn't contain enough blocks, we will fail on a tombstone block error and not get to the signature checking.

Reply:
Not sure I understand the test passes locally when I make the change to 1 block

Reply:
Oh yeah, my bad.